src/ts-transformer/utils/expression-operators.tsx

```diff
@@ -0,0 +1,20 @@
+import * as ts from 'typescript';
+
+/**
+ * Joins two expressions to a jsx expression, separated by an operator and a space
+ * Example: 'myString' + ' ' + myFunction()
+ */
+export function joinExpression(
```

Comment:
since this is for string expressions maybe call this...

```suggestion
export function joinStringLiteralExpression(
```